crates/uv-client/Cargo.toml

```diff
@@ -48,14 +48,11 @@ tracing = { workspace = true }
 url = { workspace = true }
 urlencoding = { workspace = true }
 
-# These must be kept in-sync with those used by `reqwest`.
-rustls = { version = "0.21.10" }
-rustls-native-certs = { version = "0.6.3" }
-webpki-roots = { version = "0.25.4" }
-
 [dev-dependencies]
 anyhow = { workspace = true }
-hyper = { version = "0.14.28", features = ["server", "http1"] }
+http-body-util = { version = "0.1.0" }
+hyper = { version = "1.2.0", features = ["server", "http1"] }
+hyper-util = { version = "0.1.3", features = ["tokio"] }
 insta = { version = "1.36.1" }
 os_info = { version = "3.7.0", default-features = false }
```

Comment:
@charliermarsh I noticed this got updated to `3.8.2` in the lock file, causing the error in ubuntu. This should probably be switched to `os_info = { version = "=3.7.0", default-features = false }` in the mean time as 3.8 has had some breaking changes.

Reply:
I'll do a follow-on PR after this to improve these tests and hopefully remove this dependency.

Reply:
Thank you so much.

Reply:
Updated...

Reply:
Awesome, tests passing 🎉